resolwe_bio/processes/modeling/pathway_predictor.yml

```diff
@@ -1,13 +1,16 @@
-- slug: pathway-predictor
+- slug: pathways-predictor
   name: Predict heterologous metabolic pathways
   requirements:
     expression-engine: jinja
     executor:
       docker:
         image: dddecaf/resolwe-pathway-predictor:latest
+    resources:
+      network: true
+      cores: 4
   data_name: "Predict heterologous metabolic pathways"
   version: 0.0.1
-  type: data:similarexpression
+  type: data:pathways
   persistence: TEMP
   scheduling_class: interactive
```

Comment:
Processes with ``scheduling_class: interactive`` are run in a separate queue and are killed after 30s if not finished in time. This happens in the test case already, so you probably need to remove this.